### DISCUSSION TITLE
Feature request: add more localisation options

### DISCUSSION THREAD

**Discussion by EmielRous** [GreenmaskIO/greenmask#204] [Ideas]
I'm working with a lot of data from different countries. Most of the faker functions also allow you to specify more context, like the fakerRealAddress you can add options to set the country. It would be amazing if we could pass arguments to the functions (both in the standard transformations and the template functions)

**Comment by wwoytenko** [comment]
Hi! That’s a great idea. I’ll try to implement it in the 4th quarter. I’ll create a feature request soon.

**Reply by wwoytenko** [reply]
There is feat request #206

**Reply by wwoytenko** [reply]
@EmielRous FYI
